grafana/data_source_organization_users.go

```diff
@@ -0,0 +1,62 @@
+package grafana
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DatasourceOrganizationUsers() *schema.Resource {
+	return &schema.Resource{
+		Description: `
+* [Official documentation](https://grafana.com/docs/grafana/latest/administration/manage-organizations/)
+* [HTTP API](https://grafana.com/docs/grafana/latest/http_api/org/)
+`,
+		ReadContext: dataSourceOrganizationUsersRead,
+		Schema: map[string]*schema.Schema{
+			"organization_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the Organization.",
+			},
+			"emails": {
```

Comment:
I considered calling this `user_emails` and/or `user_email_addresses`, though `emails` seems the most terse, clear, and consistent with the `email` attribute used elsewhere by the API and provider. Is that reasonable?